### PR TITLE
Robust Cult changes

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
experiment: You can examine tomes, soulstones, and construct shells to see what you can do with them.
rscadd: Releasing a shade from a soulstone now allows you to reuse that soulstone for capturing shades or souls, instead of preventing the stone's use forever if the shade dies.
imageadd: Runes now actually pulse red on failure.
rscadd: The Summon Cultist rune no longer includes the cultists invoking it in the selection menu.
tweak: Imperfect Communication renamed to Emergency Communication.
rscdel: You can no longer hide the Nar-Sie rune with a Talisman of Veiling/Revealing.
/:cl:

Runes now generate a list of invokers when invoked, and cause those invokers to speak/take damage/whatever the rune does instead of many, many for(cultist in range) loops.

### CODE REVIEW

**Comment by Robustin** [comment]
All good stuff. It's a shame we couldn't move forward on this sooner....

File: code/game/gamemodes/cult/ritual.dm [hunk 1]
```diff
@@ -144,87 +140,86 @@ This file contains the arcane tome files as well as innate cultist emergency com
...
+
 	text += "In addition to these runes, the cult has a small selection of equipment and constructs.<br><br>"
 
 	text += "<font color='red'><b>Equipment:</b></font><br><br>"
 
-	text += "<font color='red'><b>Cult Blade</b></font><br>Cult blades are a sharp weapons that, notably, cannot be used by noncultists. These blades are produced by the Rite and Talisman of Arming.<br><br>"
+	text += "<font color='red'><b>Cult Blade</b></font><br>Cult blades are sharp weapons that, notably, cannot be used by noncultists. These blades are produced by the Talisman of Arming.<br><br>"
 
-	text += "<font color='red'><b>Cult Robes</b></font><br>Cult robes are heavily armored robes. These robes are produced by the Rite and Talisman of Arming.<br><br>"
+	text += "<font color='red'><b>Cult Bola</b></font><br>Cult blola are strong bolas, useful for snaring targets. These blades are produced by the Talisman of Arming.<br><br>"
```
Comment (Man-Shroom-) [inline code/game/gamemodes/cult/ritual.dm]: >blola

is this intended?

**Comment by Zilenan91** [comment]
IT'S HAPPENING AGAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIN

**Comment by ChangelingRain** [comment]
The war never ends, Citizen.

File: code/game/gamemodes/cult/runes.dm [hunk 2]
```diff
@@ -360,46 +376,41 @@ var/list/teleport_runes = list()
...
 			rune_in_use = 0
 			return
 	visible_message("<span class='warning'>[src] pulses blood red!</span>")
 	color = rgb(126, 23, 23)
-	spawn(5)
-		color = initial(color)
-	sac(offering)
+	..()
+	sac(invokers, offering)
+	color = initial(color)
```
Comment (phil235) [inline code/game/gamemodes/cult/runes.dm]: won't the color changes here conflict with the ones in ..() ?

File: code/game/gamemodes/cult/runes.dm [hunk 3]
```diff
@@ -421,60 +432,67 @@ var/list/teleport_runes = list()
...
+				M << "<span class='warning'>Nar-Sie is already on this plane!</span>"
+			log_game("Summon Nar-Sie rune failed - already summoned")
+			return
 		//BEGIN THE SUMMONING
 		used = 1
-		for(var/mob/living/M in range(1,src))
-			if(iscultist(M))
-				M.say("TOK-LYR RQA-NAP G'OLT-ULOFT!!")
+		color = rgb(255, 0, 0)
+		..()
```
Comment (phil235) [inline code/game/gamemodes/cult/runes.dm]: same here

File: code/game/gamemodes/cult/runes.dm (hunk 2 above)
Comment (ChangelingRain) [inline code/game/gamemodes/cult/runes.dm]: ..() doesn't change color, it only changes alpha and transform. It'd conflict with fail_invoke(), but if we're calling ..() in invoke we aren't going to call fail_invoke

Comment (phil235) [inline code/game/gamemodes/cult/runes.dm]: oh, yeah, sorry, I read the wrong lines.

File: code/game/gamemodes/cult/runes.dm (hunk 3 above)
Comment (ChangelingRain) [inline code/game/gamemodes/cult/runes.dm]: see above

**Pull request merged**